.github/workflows/go.yml

```diff
@@ -66,6 +66,10 @@ jobs:
           key: ${{ runner.os }}-embedded-bins-${{ hashFiles('**/embedded-bins/**/*') }}
           restore-keys: |
             ${{ runner.os }}-embedded-bins-${{ hashFiles('**/embedded-bins/**/*') }}
+      - name: Generate manifests
+        id: gen-manifests
+        run: |
+          sudo apt-get install go-bindata
```

Comment:
What do you think about having it as a dependency (the same way we do with go-lint) to have way to pin and update versions without relying on debian maintainers?

Reply:
No, I don't

Let me guide you through what I meant.
In a Makefile we have target `lint` to run linter. It could be used either locally to run linter without even touching CI or through CI.
By having this `go get ...` code we pin the particular version of the golint to use.
I suggest do the same way for go-bindata because with go-bindata it could be flight critical to have particular version pinned.
```
golint := $(shell which golangci-lint)
ifeq ($(golint),)
golint := go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.31.0 && "${GOPATH}/bin/golangci-lint"
endif

.PHONY: lint
lint: pkg/assets/zz_generated_offsets_$(TARGET_OS).go
	$(golint) run ./...
```

Reply:
Got it, thanks for the clarification. 
I've implemented it similar to the `golangci-lint` (i.e check if installed locally, or else get via `go get ...` and use that binary)